.ci/Jenkinsfile

```diff
@@ -46,7 +46,7 @@ pipeline {
       }
     }
     stage('Test Hosts'){
-      failFast false
+      failFast true
```

Comment:
wdyt about using `parallelsAlwaysFailFast()` in the top-level `options` section? Any preference? I see it more readable being declared at the beginning of the pipeline

Reply:
No strong opinion, but I've never used `parallelsAlwaysFailFast()` but failFast, so I can change it

Reply:
I ended up using failFast (as in this PR) in https://github.com/elastic/e2e-testing/pull/429 See there rationale